addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/handler/AVMFritzBaseBridgeHandler.java

```diff
@@ -375,6 +412,14 @@ public String getThingTypeId(AVMFritzBaseModel device) {
             } else if (device.isSwitchableOutlet()) {
                 return GROUP_SWITCH;
             }
+        } else if (device instanceof DeviceModel && device.isHANFUNUnit()) {
+            List<String> interfaces = Arrays
+                    .asList(((DeviceModel) device).getEtsiunitinfo().getInterfaces().split(","));
```

Comment:
Are all of them guaranteed to be set, no NPE's hiding out here?

Reply:
No, according to the AVM documentation these elements are mandatory for HAN-FUN devices. I can add a link to a PDF file if you like but unfortunately it is German only.

Reply:
Okay fine